app/src/ui/updates/update-available.tsx

```diff
@@ -25,7 +25,8 @@ export class UpdateAvailable extends React.Component<
           An updated version of GitHub Desktop is available and will be
           installed at the next launch. See{' '}
           <LinkButton uri={this.props.releaseNotesLink}>what's new</LinkButton>{' '}
-          or <LinkButton onClick={this.updateNow}>restart now</LinkButton>.
+          or <LinkButton onClick={this.updateNow}>restart Github 
```

Comment:
Thoughts on changing this to "restart GitHub Desktop" instead?

The two minor changes are capitalizing the "H" in GitHub and I don't think the "now" is necessary (although I don't strongly feel either way.)